torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -761,6 +761,31 @@ const std::vector<std::string> functions = {
                 return grad_output / other, None
             return self / other, backward
 
+        def div_2(self, other, rounding_mode: str):
```

Comment:
Are these necessary?

Reply:
If I remove it the autodiff test fails, if I remove `assert_autodiffed` from the `OpInfo` then I instead get this error:
```
E   AssertionError: False is not true : 
E   Failure in testing nodes' autodifferentiation, one or more nodes were not expected to be autodiffed, but were found in a fused/nonfused DifferentiableGraph group. Did you intend for these nodes to be autodiffed? If so, change this test to expect autodifferention. 
E   Specifically:
E     aten::div was not expected to be in a DifferentiableGraph but was.
```
Because some sample inputs aren't autodiffed while others are.

Reply:
This goes back to my previous question, which is whether we should model this as multiple operators for testing?

Reply:
So I should have 3 `OpInfos`: no `rounding_mode`, `rounding_mode="true"` and `rounding_mode={"floor", "trunc"}`?

Reply:
That seems more straightforward than these other workarounds to me, but I could be convinced it's easier to create one OpInfo (or two, or four) if you think that's best.

Reply:
I split the `OpInfo`s up by `rounding_mode` but it was still inconsistent even for the same rounding mode. So, I think what's happening is the tensor overload is implicitly autodiff-able but the scalar version has to be defined manually.

Reply:
Interesting... let me know if it would be helpful to chat offline about the details here. I'd like to find a simple approach to address this.